services_and_applications/jupyter/overview.rst

```diff
@@ -120,40 +120,48 @@ Conda environments need to be saved in a ``/ccs`` or ``/gpfs/alpine`` directory.
 
 **Please note that GPFS is purged. Using /ccs/proj/ is recommended**
 
-Let us look at an example with creating a custom environment that has Tensorflow. 
+Let us look at an example with creating a custom environment that has RAPIDS (https://rapids.ai/start.html).
 
-At the end of this example you will have a "tensorflowenv" notebook visible in the JuptyerLab "Launcher" page.
+At the end of this example you will have a "rapids" notebook visible in the JuptyerLab "Launcher" page. We will make the RAPIDS environment persistent by 
+building it at /ccs/proj/<uid>/rapids. 
 
 
-Example: Creating a Conda environment for Tensorflow
-====================================================
+Example: Creating a Conda environment for RAPIDS
+=================================================
 
 #. From the Launcher page in JupyterLab, click on Terminal.
-#. Create a conda environment with  ``conda create -p /path/to/tensorflowenv python=3.7
-   ipykernel``.
+#. Create a conda environment with 
+   ``conda create -p /ccs/proj/<YOUR_PROJECT_ID>/<YOUR_UID>/rapids -c rapidsai -c nvidia -c conda-forge \``
+   ``-c defaults rapids-blazing=0.17 python=3.7 cudatoolkit=10.2``
+   
+   * **NOTE**: The *conda create* command above assumes you are using **CUDA 10** JuptyerLab environment.
    
    * You need to use ``-p`` method if you want your environment to persist across Jupyter
      restarts. The path can be a location in ``/ccs`` or ``/gpfs/alpine`` that is writable
      by your user.
      
-#. Activate the environment ``source activate /path/to/tensorflowenv``.
+#. Activate the environment ``source activate /ccs/proj/<YOUR_PROJECT_ID>/<YOUR_UID>/rapids``.
```

Comment:
We should add a note here saying that the user should use `source activate` and not `conda activate` even though that is what is recommended at the end of the `conda create` process. Because conda activate requires extra setup and needs modification of their bashrc file.